lib/datadog/di/instrumenter.rb

```diff
@@ -152,7 +153,28 @@ def hook_method(probe, &block)
                 serialized_entry_args: entry_args)
               rv
             else
-              super(*args, **kwargs)
+              # stop standard from trying to mess up my code
+              _ = 42
```

Comment:
Are are you able to do `# standard:disable` instead here?

Reply:
The issue is not on this line - standard is trying to lift the contents of this branch up to the parent conditional, in the process completely ruining the encapsulation of the fallback logic that is currently very clear. It wants to have a huge single conditional instead of two when the second one is nested in the `else` branch of the first one.

Reply:
It is not even deterred by my giant comment explaining the code that follows - it thinks it's perfectly fine to move that comment to one of the branches that it pertains to (rather than the entire logic block).